src/test/operators/validate_test.cpp

```diff
@@ -52,7 +65,10 @@ void OperatorsValidateTest::set_all_records_visible(Table& table) {
 }
 
 void OperatorsValidateTest::set_record_invisible_for(Table& table, RowID row, CommitID end_cid) {
-  table.get_chunk(row.chunk_id)->get_scoped_mvcc_data_lock()->end_cids[row.chunk_offset] = end_cid;
+  auto chunk = table.get_chunk(row.chunk_id);
```

Comment:
I don't like `set_record_invisible_for` - what does "for" refer to? Just `set_record_invisible` or, even better `invalidate_record`?

Reply:
Hm, I guess "for" should refer to the CommitID? But I wasn't the author of this method.

This does not proof anything, but there is a fair chance that you came up with it :D `c9fc17080f (Markus Dreseler 2017-03-23 21:59:28 +0100  67) void OperatorsValidateTest::set_record_invisible_for(Table& table, RowID row, CommitID end_cid) {`